.devcontainer/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM mcr.microsoft.com/devcontainers/typescript-node:20
+
+# Install Deno
+ENV DENO_INSTALL=/usr/local
+RUN curl -fsSL https://deno.land/install.sh | sh
```

Comment:
```suggestion
RUN curl -fsSL https://gist.githubusercontent.com/LukeChannings/09d53f5c364391042186518c8598b85e/raw/ac8cd8c675b985edd4b3e16df63ffef14d1f0e24/deno_install.sh | sh
```

Reply:
Cool! It works on my machine.

Hey @C-Dao . What do you think this way? I'd like to go with this.

Reply:
> Cool! It works on my machine.
> 
> Hey @C-Dao . What do you think this way? I'd like to go with this.

LGTM, thanks @CyberFlameGO!